File: Kotlin/ascii.kt
fun numeroParaASCII(numero: Int): String {
    if (numero < 0 || numero > 127) return "Valor invalido!"
    return numero.toChar().toString()
}

fun main() {
    print("Digite um numero (0-127): ")
    val numero = readLine()?.toIntOrNull() ?: -1
    println("ASCII: ${numeroParaASCII(numero)}")
}

File: R/ascii.R
numero_para_ascii <- function(numero) {
  if (numero < 0 || numero > 127) return("Valor invalido!")
  rawToChar(as.raw(numero))
}

numero <- as.integer(readline("Digite um numero (0-127): "))
cat("ASCII:", numero_para_ascii(numero), "\n")
